lib/constructs/attribute.js

```diff
@@ -23,19 +23,15 @@ class Attribute {
 
     const onInstance = utils.isOnInstance(this.idl, this.interface.idl);
 
-    let objName = `this`;
-    if (onInstance) { // we're in a setup method
-      objName = `obj`;
-    }
```

Comment:
Can you tell me if my understanding is correct: this isn't needed because for unforgeable methods/properties `this` always points to `obj` anyway?

Reply:
Well, yes, unless you were to use `unforgeableMethod.call`, in which case `this` will be whatever was passed to `call` as the `thisArg` parameter.